tests_e2e/playwright/README.md

```diff
@@ -0,0 +1,189 @@
+# Nebari integration testing with Playwright
+
+
+## How does it work?
+
+Playwright manages interactions with any website. We are using it to interact
+with a deployed Nebari instance and test the various integrations that are
+included.
+
+For our test suite, we utilize Playwright's synchronous API. The first task
+is to launch the web browser you'd like to test in. Options in our test suite
+are `chromium`, `webkit`, and `firefox`. Playwright uses browser contexts to
+achieve test isolation. The context can either be created by default or
+manually (for the purposes of generating multiple contexts per test in the case
+of admin vs user testing). Next the page on the browser is created. For all
+tests this starts as a blank page, then during the test, we navigate to a given
+url. This is all achieved in the `setup` method of the `Navigator` class.
+
+## Setup
+
+Install Nebari with the development requirements (which include Playwright)
+
+`pip install -e ".[dev]"`
+
+Then install playwright itself (required).
+
+`playwright install`
+
+> If you see the warning `BEWARE: your OS is not officially supported by Playwright; downloading fallback build., it is not critical.` Playwright will likely still work microsoft/playwright#15124
+
+### Create environment file
+
+Create a copy of the `.env` template file
+
+```bash
+cp .env.tpl .env
+```
+
+Fill in the newly created `.env` file with the following values:
+
+* USERNAME: Nebari username for username/password login OR Google email address or Google sign in
+* PASSWORD: Password associated with USERNAME
+* NEBARI_FULL_URL: full url path to Nebari instance, e.g. "https://nebari.quansight.dev/"
+
+## Running the Playwright tests
+
+The playwright tests are run inside of pytest using
+
+```python
+pytest tests_e2e/playwright/test_playwright.py
+```
```

Comment:
This currently doesn't work, since the test expects `test_data` to be in the current directory.